content/Foundation/Summit/2023/home.pml

```diff
@@ -31,8 +31,62 @@ Sept/Oct 2023, dates TBC
       Latest News
     </h3>
   </div>
+  <h3 class="highlight">
+   Call For Presentations for the seL4 Summit 2023
```

Comment:
Add link in the title as well. Plus, on the bullet points just before that, have: "Latest News: Call for Presentations" so that it appears in the bullet list.